contracts/TracerPerpetualSwaps.sol

```diff
@@ -204,24 +209,44 @@ contract TracerPerpetualSwaps is
         Balances.Account storage account1 = balances[order1.maker];
         Balances.Account storage account2 = balances[order2.maker];
 
+        int256 fee =
+            PRBMathUD60x18
+                .mul(uint256(quoteChange), uint256(feeRate.toInt256()))
+                .toInt256();
+
         /* TODO: handle every enum arm! */
         if (order1.side == Perpetuals.Side.Long) {
             // user 1 is long. Increase base, decrease quote
-            account1.position.quote = account1.position.quote - quoteChange;
+            account1.position.quote =
+                account1.position.quote -
+                quoteChange -
+                fee;
             account1.position.base = account1.position.base + _fillAmount;
 
             // user 2 is short. Increase quote, decrease base
-            account2.position.quote = account2.position.quote + quoteChange;
+            account2.position.quote =
+                account2.position.quote +
+                quoteChange +
+                fee;
```

Comment:
I think this should also be subtracting the fee here

Reply:
Are you sure? Since the user is going short, we would want to decrease their margin a bit.